### ISSUE TITLE
Suggestions RE balanced audio

### ISSUE THREAD

**Issue by privatezero** [amiaopensource/apex_video_kit_docs#8]
I would suggest a few changes to the [section](https://github.com/amiaopensource/apex_video_kit_docs/blob/suggested_edits/video_kit_elements_handout.md#direct-box) about direct boxes and balanced audio. Black Magic devices can be set for either pro level or consumer level inputs (at least when using the Black Magic capture software - have to see how vrecord interacts with this) so perhaps just a note to double check settings would be helpful?

The difference in output levels between pro (using XLR) and consumer equipment (using RCA) really is what has to be watched out for rather than the balanced vs unbalanced signals, which just help protect against noise and interference in the cable rather than making the audio inherently better. I don't see much of an advantage in converting an unbalanced consumer level signal to a balanced pro level signal if the A/D can be configured to accept the consumer level signal as is. Theoretically you would have the same odds of your unbalanced cables picking up noise on their way into the direct box as you would going straight into the A/D converter.

**Comment by pvizner** [comment]
Hi Andrew! Thanks for your suggestion, I see your point. Yes, the blackmagic can take both balanced and unbalanced, but some AD converters don't (some old AJA converters for example). Since this kit is meant to be educational, we want people to be aware of all the issues they can find when setting up a station, hence the use of the DI box. I know there's other ways of converting from unbalanced to balanced, such as using a mixer. Anyway, there's tons of options, but the idea is that people get to think about them and understand the concepts. But you're right, we'll make a note in the documentation so this is clear. Thanks again!

**Comment by privatezero** [comment]
Yeah, it is a bit of a sticky issue in terms of the issues that can crop up! And additionally complicated by A/D converters that only have RCA in like the cheaper Blackmagic boxes!

Maybe just some wording about trying to stick with XLR -> XLR or RCA -> RCA when possible and the options that are available if changing the signal is necessary? As well as a warning to check settings on the A/D, because I know that at least Blackmagic hides things in the menu and they are very easy to overlook!
